minio/nginx/conf.d/minio.conf

```diff
@@ -0,0 +1,76 @@
+server {
```

Comment:
Minor but make sure indentation are correct.

Reply:
Maybe I was not clear enough but I was speaking about the whole file :)

Reply:
Please see: https://github.com/MariaDB/buildbot/pull/617/commits/47dba51394a947012cf3357a23270a6b64205a5b
I used: https://github.com/slomkowski/nginx-config-formatter